go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/montanaflynn/stats v0.0.0-20171201202039-1bf9dbcd8cbe
 	github.com/nats-io/nats-server/v2 v2.7.5-0.20220415000625-a6b62f61a703
 	github.com/nats-io/nats.go v1.21.0
-	github.com/numaproj/numaflow-go v0.2.4
+	github.com/numaproj/numaflow-go v0.2.5
```

Comment:
let's cut `0.3.0` for numaflow-go

Reply:
may I merge https://github.com/numaproj/numaflow-go/pull/26 and cut 0.3.0?